fiftyone/utils/huggingface.py

```diff
@@ -302,23 +341,34 @@ def _get_fiftyone_version(self, kwargs):
 """
 
 
+def _validate_fiftyone_version(version_string):
+    ## match to "a.b.c" where a, b, c are integers
+    return bool(re.match(r"\d+\.\d+\.\d+", version_string))
+
+
 def _populate_config_file(
     config_filepath,
     dataset,
     dataset_type=None,
     description=None,
     license=None,
     tags=None,
+    min_fiftyone_version=None,
 ):
     config_dict = {
         "name": dataset.name,
         "format": dataset_type.__name__,
-        "fiftyone": {
-            "version": f">={foc.VERSION}",
-        },
         "tags": tags,
     }
 
+    if min_fiftyone_version is not None:
+        if not _validate_fiftyone_version(min_fiftyone_version):
```

Comment:
I would recommend removing this validation as version numbers could actually contain letters and stuff (not usual, but could, eg release candidates).

How about just providing an example in the docstring?

```
        min_fiftyone_version (None): the minimum version of FiftyOne required,
            if applicable. For example ``"0.23"``
```